x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/InferenceIndex.java

```diff
@@ -84,6 +85,50 @@ public static XContentBuilder mappings() {
                 .startObject("properties")
                 .endObject()
                 .endObject()
+                .startObject("chunking_settings")
+                .field("dynamic", "false")
+                .startObject("properties")
+                .startObject("strategy")
+                .field("type", "keyword")
+                .endObject()
+                .endObject()
+                .endObject()
+                .endObject()
+                .endObject()
+                .endObject();
+        } catch (IOException e) {
+            throw new UncheckedIOException("Failed to build mappings for index " + INDEX_NAME, e);
+        }
+    }
+
+    public static XContentBuilder mappingsV1() {
+        try {
+            return jsonBuilder().startObject()
+                .startObject(SINGLE_MAPPING_NAME)
+                .startObject("_meta")
+                .field(SystemIndexDescriptor.VERSION_META_KEY, 1)
+                .endObject()
+                .field("dynamic", "strict")
+                .startObject("properties")
+                .startObject("model_id")
+                .field("type", "keyword")
+                .endObject()
+                .startObject("task_type")
+                .field("type", "keyword")
+                .endObject()
+                .startObject("service")
+                .field("type", "keyword")
+                .endObject()
+                .startObject("service_settings")
+                .field("dynamic", "false")
+                .startObject("properties")
+                .endObject()
+                .endObject()
+                .startObject("task_settings")
+                .field("dynamic", "false")
+                .startObject("properties")
+                .endObject()
```

Comment:
Looks like an empty properties object?

```suggestion
```

Reply:
This is copied from our "old mappings" ([here](https://github.com/elastic/elasticsearch/blob/main/x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/InferenceIndex.java#L84)). I'm not exactly sure why we have it but I figured it's best to just keep it as is. If you think it's critical to remove it, I can do that though.

Reply:
Because these are new mappings we can correct the mistakes in the v1 mappings. We have plenty of test coverage so if anything does go wrong the tests will find it

Reply:
Sure, I'll remove the empty properties from all the mappings in this file.

Reply:
Oh so it is required. Good to know thanks